README.md

```diff
@@ -2,3 +2,17 @@ ES6 Fiddle Web
 ===
 
 [![Build Status](https://travis-ci.org/jmcriffey/es6-fiddle-web.png?branch=master)](https://travis-ci.org/jmcriffey/es6-fiddle-web)
+
+#### Online Version
+[ES6 Fiddle Web](http://www.es6fiddle.net/)
+
+#### How to install locally ?
+1. clone the repo.
+2. 
+```> cd es6-fiddle-web ```
```

Comment:
If I update the environment variable that the url is read from, point 4 should read 'add ES6FIDDLE_DB_URL to your environment' or something similar.